stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/internal/LiveData.kt

```diff
@@ -34,3 +41,19 @@ internal fun <P, Q> Pair<LiveData<P>, LiveData<Q>>.observeTogether(
         block(this.first.value, it)
     }
 }
+
+/**
+ * Adds Flow as a Source to this MediatorLiveData
+ */
+internal fun <T, S> MediatorLiveData<T>.addFlow(
+    context: CoroutineContext = EmptyCoroutineContext,
```

Comment:
In this case if we pass in a SupervisorJob that has only a Job as a context parameter, we're not not doing anything, we should pass in the correct context that uses a valid dispatcher etc.

Reply:
I think that should be enough - sorry for the misinterpretation of what the SupervisorJob "constructor" was doing

Reply:
Sure, no worries, man 👌